.github/workflows/pull-request.yml

```
@@ -201,12 +204,29 @@ jobs:
         run: docker compose build
       - name: Run tests
         run: docker compose up --exit-code-from api
+      - name: Copy coverage from container
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:11: Double quote to prevent globbing and word splitting

(shellcheck)